main.go

```diff
@@ -32,48 +96,317 @@ func main() {
+						if err != nil {
+							return err
+						}
+						w := tabwriter.NewWriter(os.Stdout, 0, 0, 1, ' ', tabwriter.AlignRight|tabwriter.Debug)
+						fmt.Fprintln(w, "Product_id\tName\tPrice\tSku\t")
+						fmt.Fprintf(w, "%d\t%s\t%v\t%v\t\n", productId, name, price, sku)
+						w.Flush()
+					} else {
+						insertStatement := "INSERT INTO Products (name, price) VALUES (?, ?)"
+						res, err := db.Exec(insertStatement, name, price)
+						if err != nil {
+							return err
+						}
+						productId, err := res.LastInsertId()
+						if err != nil {
+							return err
+						}
+						w := tabwriter.NewWriter(os.Stdout, 0, 0, 1, ' ', tabwriter.AlignRight|tabwriter.Debug)
+						fmt.Fprintln(w, "Product_id\tName\tPrice\t")
+						fmt.Fprintf(w, "%d\t%s\t%v\t\n", productId, name, price)
+						w.Flush()
+					}
+					return nil
+
+				},
+			},
+			{
+				Name:  "create-order",
+				Usage: "Creates a new order to go in the order database, must specify customer_id and product_id",
+				Action: func(cCtx *cli.Context) error {
+					if len(os.Args) != 4 {
+						return errors.New("Must specify product_id and customer_id")
+					}
+					pID, err := strconv.Atoi(os.Args[2])
+					if err != nil {
+						return err
+					}
+					cID, err := strconv.Atoi(os.Args[3])
+					if err != nil {
+						return err
+					}
+					insertStatement := "INSERT INTO Orders (customer_id, product_id) VALUES (?, ?)"
+					res, err := db.Exec(insertStatement, cID, pID)
+					if err != nil {
+						return err
+					}
+					orderID, err := res.LastInsertId()
+					if err != nil {
+						return err
+					}
+
+					w := tabwriter.NewWriter(os.Stdout, 0, 0, 1, ' ', tabwriter.AlignRight|tabwriter.Debug)
+					fmt.Fprintln(w, "Order_id\tProduct_id\tCustomer_id\t")
+					fmt.Fprintf(w, "%v\t%v\t%v\t\n", orderID, pID, cID)
+					w.Flush()
+
+					return nil
+				},
+			},
+			{
+				Name:  "show-products",
+				Usage: "Shows the proudcts from the products database, optional flag name to filter by name",
 				Flags: []cli.Flag{
 					&cli.StringFlag{
 						Name:  "name",
-						Usage: "the name we want to display",
+						Usage: "the name of the order",
 					},
 				},
 				Action: func(cCtx *cli.Context) error {
-					name := "<none>"
 					if cCtx.String("name") != "" {
-						name = cCtx.String("name")
+						name := cCtx.String("name")
+						w := tabwriter.NewWriter(os.Stdout, 0, 0, 1, ' ', tabwriter.AlignRight|tabwriter.Debug)
+
+						rows, err := db.QueryContext(ctx, "SELECT * FROM PRODUCTS WHERE Name LIKE CONCAT('%', ?, '%')", name)
+						if err != nil {
+							return err
+						}
+						defer rows.Close()
+
+						productPrintHelper(rows, w)
+						w.Flush()
+					} else {
+
+						rows, err := db.QueryContext(ctx, "SELECT * FROM PRODUCTS")
+						w := tabwriter.NewWriter(os.Stdout, 0, 0, 1, ' ', tabwriter.AlignRight|tabwriter.Debug)
+						if err != nil {
+							return err
+						}
+						defer rows.Close()
+						productPrintHelper(rows, w)
```

Comment:
Looks like ya forgot to handle the error returned by productPrintHelper

Reply:
there's quite a few instances of these missing error handlers for the *PrintHelper methods; just commenting on this one